script/Arcueid/Day 2/02_00_ARC02_1.txt

```diff
@@ -30,13 +30,15 @@ You do not dream.
 }
 [sha:e3d9e5db51606e5e74f98b9801166a8ae326489f]{
 -- Page 3, Offset 2602.
+// i don't like this whole 'even now' thing, but i dont exactly know what to do with it
```

Comment:
reversing the phrase might make it sound a little more natural: "Even now, I'm not here."